src/Notifier.js

```diff
@@ -123,6 +123,26 @@ var Notifier = {
         return global.Notification.permission == 'granted';
     },
 
+    isPermissionDefault: function() {
+        if (!this.supportsDesktopNotifications()) return false;
+        return global.Notification.permission == 'default';
+    },
+
+    showToolbar: function() {
```

Comment:
It's used by client to know weather or not to show the toolbar, I'll change it to `shouldShowToolbar()` though
https://github.com/vector-im/vector-web/pull/1237